client/js/components/GameView/GameOverlays.vue

```diff
@@ -104,6 +113,25 @@ export default {
       }
       return mayCounter;
     },
+    showWaitingForOpponentToDiscardJackFromDeck() {
+      return (
+        this.waitingForOpponentToPlayFromDeck &&
+        this.topTwoCardsCantBePlayed &&
+        (this.opponentPointTotal === 0 || this.opponentQueenCount > 0)
+      );
+    },
+    showWaitingForOpponentToPlayFromDeck() {
+      return this.waitingForOpponentToPlayFromDeck && !this.topTwoCardsCantBePlayed;
+    },
+    topTwoCardsCantBePlayed() {
+      return this.topCard.rank === 11 && (!this.secondCard || this.secondCard.rank === 11);
+    },
+    topCard() {
+      return this.$store.state.game.topCard;
+    },
+    secondCard() {
+      return this.$store.state.game.secondCard;
+    },
```

Comment:
These two can be done with `mapState()` next to the `waitingForOpponent...` stuff